vendor/brave-ios/Ledger/BATBraveLedger.mm

```diff
@@ -551,10 +551,12 @@ - (void)fetchGrants:(const std::string &)lang paymentId:(const std::string &)pay
   ledger->FetchGrants(lang, paymentId);
 }
 
-- (void)solveGrantCaptchWithPromotionId:(NSString *)promotionId solution:(NSString *)solution
+- (void)solveGrantCaptchWithPromotionId:(NSString *)promotionId solution:(NSString *)solution completion:(void (^)(BATResult result, ledger::GrantPtr grant))completion
 {
   ledger->SolveGrantCaptcha(std::string(solution.UTF8String),
-                            std::string(promotionId.UTF8String));
+  std::string(promotionId.UTF8String), ^(ledger::Result result, ledger::GrantPtr grant) {
+    // To be implemented
+  });
 }
 
 - (void)onGrantFinish:(ledger::Result)result grant:(ledger::GrantPtr)grant
```

Comment:
Contents of this method below should be moved to callback of `ledger->SolveGrantCaptcha` above and then this method should then be removed